lib/Doctrine/ORM/UnitOfWork.php

```diff
@@ -3036,83 +3019,6 @@ public function size()
     }
 
     /**
-     * Gets the EntityPersister for an Entity.
-     *
-     * @param string $entityName The name of the Entity.
-     *
-     * @return \Doctrine\ORM\Persisters\Entity\EntityPersister
-     */
-    public function getEntityPersister($entityName)
```

Comment:
The method should be kept public for backward compatibility (for extensions hooking into Doctrine)